README.md

```diff
@@ -14,3 +18,13 @@ make html
 ```
 
 Afterwards, open \_build/index.html in a browser.
+
+### Installation
+
+`albatross` has not yet been hosted on PyPI, so for now, install from Github:
+
+`pip install git+https://github.com/camirmas/albatross`
```

Comment:
This Github packages service is new to me, but I don't see Python in the list of supported languages for this service, probably because PyPI is so ubiquitous. Also, I'm not a Conda user, but I think `pip` is ok with doing it this way, it works on my machine but I'd be curious to know if it works for you.